metricbeat/module/windows/_meta/config.yml

```diff
@@ -1,2 +1,8 @@
 - module: windows
-  period: 60s
+  metricsets: ["perfmon"]
```

Comment:
This is not enabled by default so it should not be here.

Reply:
I commented it out